.github/workflows/ci.yml

```
@@ -50,6 +54,18 @@ jobs:
       - uses: julia-actions/julia-uploadcoveralls@v0.1
         continue-on-error: true
 
+  Aqua:
+      name: Aqua tests
+      runs-on: ubuntu-latest
+      steps:
+          - uses: actions/checkout@v2
+          - uses: julia-actions/setup-julia@v1
+            with:
+                version: '1'
+          - uses: julia-actions/julia-buildpkg@v1
+          - run: julia -e 'using Pkg; Pkg.add("Aqua")'
+          - run: julia --project=@. -e 'using Aqua, UnitfulAstro; Aqua.test_all(UnitfulAstro, ambiguities=false)'
```

Let's use some colours :wink: And also the same project for both commands?
```suggestion
          - run: julia --project=@. --color=yes -e 'using Pkg; Pkg.add("Aqua")'
          - run: julia --project=@. --color=yes -e 'using Aqua, UnitfulAstro; Aqua.test_all(UnitfulAstro, ambiguities=false)'
```

---

Uhm, the environment was wrong? :thinking: